var/spack/repos/builtin/packages/povray/package.py

```diff
@@ -147,3 +147,9 @@ def configure_args(self):
         extra_args.append('--without-x')
 
         return extra_args
+
+    def test(self):
+        povs = find(self.prefix.share, 'biscuit.pov')[0]
+        copy(povs, '.')
+        self.run_test('povray', options=['biscuit.pov'])
```

Comment:
It would be more helpful to add a `purpose` for the test so it gets output to the test log when the test is run.